external/storm-hdfs/src/main/java/org/apache/storm/hdfs/bolt/HdfsBolt.java

```diff
@@ -44,6 +49,25 @@
     private transient FSDataOutputStream out;
     private RecordFormat format;
     private long offset = 0;
+    private static String defaultSourceDir = "/tmp/source";
+    private static String defaultDestDir = "/tmp/dest";
+    private static String defaultFileExtension = ".txt";
+
+
+    public HdfsBolt() {
```

Comment:
I'm not sure a no argument constructor makes a lot of since.  I think you need to specify at a minimum the destination directory.